server/src/main/java/org/elasticsearch/search/aggregations/bucket/sampler/random/RandomSamplerAggregationBuilder.java

```diff
@@ -163,7 +169,7 @@ public String getType() {
 
     @Override
     public Version getMinimalSupportedVersion() {
-        return Version.V_8_1_0;
+        return Version.V_8_2_0;
```

Comment:
What's the policy around setting this WRT experimental features?  I'm not totally sure how this is used.

Reply:
@not-napoleon I think the main use is failing early on CCS? Regardless, its a required override and to serialize, we need at least 8.2.0 :).